### PR TITLE
HARP-6727: Fix filtering of geometry kinds in SimpleTileGeometryManager

### PR DESCRIPTION
**Pull request by StefanDachwitz**
Signed-off-by: Stefan Dachwitz <StefanDachwitz@users.noreply.github.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: @here/harp-mapview/lib/geometry/TileGeometryLoader.ts [hunk 1]
```diff
@@ -183,8 +198,19 @@ export class SimpleTileGeometryLoader implements TileGeometryLoader {
...
+        // First time this tile is handled:
+        if (decodedTile === undefined && tile.decodedTile !== undefined) {
+            decodedTile = this.setDecodedTile(tile.decodedTile);
+            TileGeometryCreator.instance.processTechniques(
+                decodedTile,
+                enabledKinds,
+                disabledKinds
+            );
+
+            this.setDecodedTile(decodedTile);
```
Comment (okanishchev) [inline @here/harp-mapview/lib/geometry/TileGeometryLoader.ts]: Is it necessary to do setDecodedTile twice? Can't we just processTechniques on tile.decodedTile, and set it afterwards?

File: @here/harp-mapview/lib/geometry/TileGeometryCreator.ts [hunk 2]
```diff
@@ -197,9 +200,59 @@ export class TileGeometryCreator {
      */
     createAllGeometries(tile: Tile, decodedTile: DecodedTile) {
         tile.clear();
+
+        const filter = (technique: Technique): boolean => {
+            return !(technique.enabled === false);
```
Comment (okanishchev) [inline @here/harp-mapview/lib/geometry/TileGeometryCreator.ts]:
```suggestion
            return technique.enabled !== false;
```

File: @here/harp-mapview/lib/geometry/TileGeometryLoader.ts (hunk 1 above)
Comment (StefanDachwitz) [reply @here/harp-mapview/lib/geometry/TileGeometryLoader.ts]: Missed that, fixed it.

File: @here/harp-mapview/lib/geometry/TileGeometryCreator.ts (hunk 2 above)
Comment (StefanDachwitz) [reply @here/harp-mapview/lib/geometry/TileGeometryCreator.ts]: Changed that

**Pull request merged**